tests/js/firstDiscoveryEditorTests.js

```diff
@@ -34,20 +34,48 @@ https://github.com/gpii/universal/LICENSE.txt
         return builder.options.assembledPrefsEditorGrade;
     };
 
-    jqUnit.asyncTest("Initialization", function () {
+    gpii.tests.verifyStates = function (that, currentPanelNum, panel1Visible, panel2Visible, panel3Visible, backVisible, nextVisible) {
```

Comment:
rather than having direct arguments for which panels are visible, why not take in an object like.

```json
{
    visible: ["selector1", "selecto2"],
    hidden: ["selector3]
}
```

You can loop through this structure to construct your various visibility checks.

Also you should make use of the jqUnit.isVisible and jqUnit.notVisible assertions.